### DISCUSSION TITLE
What version of stable diffusion is this repo using?  Seed not yielding same results on other stable diffusion sites

### DISCUSSION THREAD

**Discussion by Kaden-Dippe** [nateraw/stable-diffusion-videos#171] [Q&A] [answered]
I'm new to stable diffusion, so please forgive me if this is a silly question.

When I use the same prompt and seed on a different site that uses stable diffusion, such as https://replicate.com/stability-ai/stable-diffusion, I'm not able to get the same image.

Is this because they are using a different noise generator to create the initial image? Is there a standard initial noise generator function used in the stable diffusion process?

**Comment by nateraw** [comment, marked as answer]
We are using the version that you specify when initializing the pipeline. Right now, the readme shows examples for [SD v1.4](https://huggingface.co/CompVis/stable-diffusion-v1-4). 

> When I use the same prompt and seed on a different site that uses stable diffusion, such as https://replicate.com/stability-ai/stable-diffusion, I'm not able to get the same image.

This is probably because you aren't running with the same hardware as them. Different hardware (GPU type) produces different results, even with the same seed. Unfortunately, as far as I know, there's nothing we can do to prevent this. 

If you use the same hardware w/ same seed, you should get the same results.
